specification/paths/Collections-collection_id.json

```diff
@@ -93,7 +93,8 @@
                           "register": {
                             "type": "boolean",
                             "example": true,
-                            "description": "Indicates whether the collection should be registered with the carrier. Further updates are no longer possible after a successful registration."
+                            "description": "Indicates whether the collection should be registered with the carrier. Further updates are no longer possible after a successful registration.",
+                            "readOnly": false
```

Comment:
Let's revert this and instead add `"readOnly": true` to the request body of the `post` path